flink-core/src/main/java/org/apache/flink/core/execution/JobClient.java

```diff
@@ -39,6 +39,12 @@
 	 */
 	JobID getJobID();
 
+	/**
+	 * Return a duplicated {@link JobClient} of this one. The duplicated one will have its
+	 * own lifecycle.
```

Comment:
Maybe we should add that this also requires closing the returned job client, to be very clear.

Reply:
👌